megazords/ios-rust/README.md

```diff
@@ -64,6 +64,8 @@ To add a new crate to the distribution:
 For testing out local changes in a consuming app, you can:
 
 * Take a local checkout of https://github.com/mozilla/rust-components-swift.
+> Note: `rust-components-swift` contains a submodule of `application-services`, if you're testing any code changes you'll need
```

Comment:
I don't quite understand this sentence:

* "if you're testing any code change" - isn't that implied here? ie, what else would I be doing if I'm reading the section that starts with "For testing out local changes"? (That's a genuine question - maybe testing something that's not a "code change"?) Or maybe you mean code changes in `rust-components-swift` vs code changes in `application-services`?

* Maybe clarify what you mean by "push/publish your branch" - presumably the person making a change is reading this because they want to test their changes *before* pushing anything, so maybe clarify how they would do that?

Reply:
Yeah I'll definitely clean this up -- the issue is that it's actually not possible to fully test locally, swift packages need to be consumed via a repository. What tripped me up on this was that I had some changes on application-services side and built the `.xcframework` correctly but when running `./make-tag` in `rust-components-swift` it still tries to use the submodule rather than my local checkout. 

Thanks for flagging this, I think adjust the docs to make this read easier where the specific problem lays